applications/serial_lte_modem/src/slm_trap_macros.h

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright (c) 2023 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#ifndef SLM_TRAP_MACROS_
+#define SLM_TRAP_MACROS_
```

Comment:
```suggestion
#ifndef SLM_TRAP_MACROS_H
#define SLM_TRAP_MACROS_H
```

It is more common to have full filename in uppercase.
https://en.wikipedia.org/wiki/Include_guard